mustang_codegen/build.yaml

```diff
@@ -15,8 +15,7 @@ builders:
     runs_before: [ 'mustang_codegen:built_value' ]
     defaults: {
       generate_for: {
-        exclude: [ '**/models/*.model.dart', '**/models/*.model.g.dart' ],
-        include: [ '**/models/*.dart' ]
+        include: [ 'lib/src/models/*.dart' ]
```

Comment:
wondering how it has been working all this time 😂 
`exclude: [ '**/models/*.model.dart', '**/models/*.model.g.dart' ]` should have caused issues while generating

Reply:
I think build runner does something like. `Include - Exclude`. Since this config is for model generator, we don't need .model.dart or .mode.g.dart files